libgadget/stats.c

```diff
@@ -73,7 +73,7 @@ struct state_of_system compute_global_quantities_of_system(void)
 
         if(P[i].Type == 0)
         {
-            struct UVBG uvbg = get_local_UVBG(redshift, P[i].Pos, All.CurrentParticleOffset);
+            struct UVBG uvbg = get_local_UVBG(redshift, P[i].Pos, CurrentParticleOffset);
```

Comment:
Is CurrentParticleOffset related to the smearing (randomShift)?

Reply:
Yes - the local UVBG needs to use the real physical position, not the shifted one.